docs/topics/vscode-extension-resolving-issues.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-resolving-issues_{context}']
+
+= Resolving issues with the {ProductName} extension
+
+You can resolve issues by using a Quick Fix or by editing the code yourself.
```

Comment:
If you have not defined Quick Fix before, you should define it here.

Remove "yourself".